src/core/public/doc_links/doc_links_service.ts

```diff
@@ -348,6 +348,7 @@ export class DocLinksService {
         maps: {
           guide: `${KIBANA_DOCS}maps.html`,
           importGeospatialPrivileges: `${KIBANA_DOCS}import-geospatial-data.html#import-geospatial-privileges`,
+          gdalTutorial: `${ELASTIC_WEBSITE_URL}blog/how-to-ingest-geospatial-data-into-elasticsearch-with-gdal`,
```

Comment:
Even though doc-links is not yet available server-side, adding the link here so it gets validated on release. This is the same approach we've taken with other data-integrations (e.g. language clients).